app/services/email_checker.rb

```diff
@@ -56,7 +56,8 @@ def compute_error
   # rubocop:enable Metrics/MethodLength
 
   def override_sendgrid?
-    @override_sendgrid
+    @override_sendgrid ||
+      ENV['ENABLE_SENDGRID_VALIDATIONS'].try(:downcase) != 'true'
```

Comment:
I find this syntax confusing and this ENV variable needs to be added to the readme.